storage-model/src/main/java/no/unit/nva/publication/storage/model/Message.java

```diff
@@ -61,9 +67,20 @@ public static Message simpleMessage(UserInstance sender,
                    .withResourceTitle(extractTitle(publication))
                    .withIsDoiRequestRelated(false)
                    .withCreatedTime(clock.instant())
+                   .withIdentifier(messageIdentifier)
+                   .withId(messageId(messageIdentifier))
```

Comment:
Id will be removed from the DAO and will only be inserted in DTOs  in future PR.